src/DI/ContainerBuilder.php

```diff
@@ -174,24 +168,21 @@ public function getAliases()
 
 
 	/**
+	 * @param  string[]
 	 * @return self
 	 */
-	public function setClassName($name)
```

Comment:
I see it's WIP, and you were probably gonna solve it anyway, but just so it doesn't disappear - removing this is a BC Break.